untitled/main/com/aca/homework/week12/stopwatch/StopWatch.java

```diff
@@ -0,0 +1,26 @@
+package com.aca.homework.week12.stopwatch;
+
+
+public class StopWatch {
+
+    private final DisplayType type;
+    protected long timer;
+
+    public StopWatch(DisplayType type) {
+        this.type = type;
+    }
+
+    public void start() {
+        long startMillis;
+        while (true) {
+            System.out.println(type.displayFormat(timer));
+            startMillis = System.currentTimeMillis();
+            while (System.currentTimeMillis() - startMillis < 10) ;
+            timer += System.currentTimeMillis() - startMillis ;
```

Comment:
isn't it better to write timer = System.System.currentTimeMillis() - startMethodInvocationMillis?